libs/utils/executor.py

```diff
@@ -512,6 +512,12 @@ def _wload_run(self, exp_idx, experiment):
         logging.debug(r'%14s - out_dir [%s]', 'Executor', experiment.out_dir)
         os.system('mkdir -p ' + experiment.out_dir)
 
+        # ANDROID: Stop all the Android system services that introduce noise
+        pause_android_services = tc.get("pause_android_services", True)
```

Comment:
Maybe it's worth to use one of the features flags instead of adding a new configuration option?
I mean something similar to what we do here:
https://github.com/ARM-software/lisa/blob/master/libs/utils/env.py#L853